Makefile

```diff
@@ -112,6 +112,9 @@ PART_SPEC_=efi conf imga
 PART_SPEC_rpi=boot conf imga
 PART_SPEC=$(PART_SPEC_$(findstring rpi,$(HV)))
 
+#parallels settings
+PARALLELS_UUID="{5fbaabe3-6958-40ff-92a7-860e329aab41}"
```

Comment:
Is this a fixed UUID? Is it a random UUID? Is it somehow related to the Parallels License? (IOW it won't work for anyone with a different license)?

Reply:
Yes, it's predefined. According to Parallels Disk Format https://github.com/qemu/qemu/blob/595123df1d54ed8fbab9e1a73d5a58c5bb71058f/docs/interop/prl-xml.txt  It is predefined GUID and related to <StorangeData> section of DiskDescriptor.

```
The software may traverse snapshots from child to parent using <ParentGUID>
field as reference. ParentGUID of root snapshot is
{00000000-0000-0000-0000-000000000000}. There should be only one root
snapshot. Top snapshot could be described via two ways: via TopGUID child
element of the Snapshots element or via predefined GUID
{5fbaabe3-6958-40ff-92a7-860e329aab41}. If TopGUID is defined, predefined GUID is
interpreted as usual GUID. All snapshot images (except Top Snapshot) should be
opened read-only. There is another predefined GUID,
BackupID = {704718e1-2314-44c8-9087-d78ed36b0f4e}, which is used by original and
some third-party software for backup, QEMU and other software may operate with
images with GUID = BackupID as usual, however, it is not recommended to use this
GUID for new disks. Top snapshot cannot have this GUID.
```

Reply:
Great -- then please add this blurb + link to the comment above @dautovri